packages/babel-plugin-proposal-decorators/test/fixtures/regression/7030/expected.js

```diff
@@ -0,0 +1,41 @@
+function _instanceof(left, right) { if (right != null && typeof Symbol !== "undefined" && right[Symbol.hasInstance]) { return right[Symbol.hasInstance](left); } else { return left instanceof right; } }
```

Comment:
non-blocker: should we standardize on using helpers or not?

Reply:
I usually don't use external helpers so that if I later add an `exec.js` test it works without modifying `options.json`

Reply:
Ok I was just thinking in general it makes everything easier to understand if we did, and then we would just need to make sure we test those separately. Or we need to fix that workflow